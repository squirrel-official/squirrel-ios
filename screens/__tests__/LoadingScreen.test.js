/**
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */

import { render } from '@testing-library/react-native';
import React from 'react';

import LoadingScreen from '../LoadingScreen';

describe('LoadingScreen', () => {
	it('should render correctly', () => {
		const { toJSON } = render(
			<LoadingScreen />
		);

		expect(toJSON()).toMatchSnapshot();
	});
});
