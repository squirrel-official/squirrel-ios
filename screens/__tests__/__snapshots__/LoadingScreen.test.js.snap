// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen should render correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#101010",
      "flex": 1,
    }
  }
>
  <Image
    fadeDuration={0}
    onLoad={[Function]}
    source={1}
    style={
      Object {
        "flex": 1,
        "height": undefined,
        "resizeMode": "contain",
        "width": undefined,
      }
    }
  />
  <ActivityIndicator />
</View>
`;
